src/main/io/gps.c

```diff
@@ -363,6 +369,33 @@ void gpsInitNmea(void)
 #endif // USE_GPS_NMEA
 
 #ifdef USE_GPS_UBLOX
+static void UBLOX_sendByteCK(const uint8_t data, uint8_t *ck_a, uint8_t *ck_b)
+{
+    *ck_a += data;
+    *ck_b += *ck_a;
+    serialWrite(gpsPort, data);
+}
+
+static void UBLOX_sendDataCK(const uint8_t *data, uint8_t len, uint8_t *ck_a, uint8_t *ck_b)
```

Comment:
What do `ck_a` and `ck_b` mean? This code will be much more maintainable if indicative names are used.

Reply:
It's for the checksum, these names comes straight from the ublox doc, and the same variable names are used in the receiving code. But yes, I agree it's time to start using clearer conventions in this particular file.